svm/Cargo.toml

```diff
@@ -26,6 +26,7 @@ solana-log-collector = { workspace = true }
 solana-measure = { workspace = true }
 solana-metrics = { workspace = true }
 solana-program-runtime = { workspace = true }
+solana-runtime-transaction = { workspace = true }
```

Comment:
@buffalojoec iirc you were trying to keep dependencies to a minimum here. Any thoughts?

Reply:
probably only other option to achieve what we need is a separate crate, so likely adding a dependency either way.

Reply:
adding this crate is to continue provide `process_compute_budget_instructions()` to SVM. And the only place SVM uses it is at `validate_transaction_fee_payer()` to calculate ComputeBudgetLimits *inline*. It is on the list of redundant calls to `process_compute_budget_instructions()` to be eliminated. (Instead, cached `ComputeBudgetLimits` (or equivalent) should be reused.) At that point, `solana-runtime-transaction` will be removed from deps.